tools/maven/suppressions.xml

```diff
@@ -47,7 +47,7 @@ under the License.
 
 	    <!-- Have to use guava directly -->
 	    <suppress
-			files="OverConvertRule.java|CustomizedConvertRule.java"
+			files="OverConvertRule.java|InConverter.java"
```

Comment:
we can also disable the check locally without bothering the rest of the project via something like...
```
    // CHECKSTYLE.OFF: MethodName
    public static ApiExpression $(String name) {
        return new ApiExpression(unresolvedRef(name));
    }
    // CHECKSTYLE.ON: MethodName
```

Reply:
Hm, placing such a comment around an import causes other checkstyle rules to break, and my IDE just removes it upon saving.

Reply:
ok, then we leave as it is